### PR TITLE
Fix broken public office info

### PR DESCRIPTION
**Pull request by EjPlatzer**
The public profile of any user with Office Info is currently broken in develop because the code is passing an object as a React child. The fix is to pass strings instead.

I also fixed all the linting warnings because of unused code and imports.

### CODE REVIEW

**Review by amos-cha — APPROVED** [review]
Thanks for the fix!

File: src/components/Profile/components/OfficeInfoList/index.jsx [hunk 1]
```diff
@@ -110,24 +104,9 @@ const OfficeInfoList = ({
...
-          href="https://go.gordon.edu/general/myaccount.cfm"
-          target="_blank"
-          rel="noopener noreferrer"
-          style={{ color: gordonColors.primary.blue }}
-        >
-          here
-        </a>
-      </Typography>
-    ) : null;
-
```
Comment (amos-cha) [inline src/components/Profile/components/OfficeInfoList/index.jsx:130]: Should we be removing the update?

Comment (amos-cha) [reply src/components/Profile/components/OfficeInfoList/index.jsx:130]: (nevermind, this was resolved by @antoniavonto )

Comment (russtuck) [reply src/components/Profile/components/OfficeInfoList/index.jsx:130]: Antonia confirmed that this is old and unused code.

**Review by russtuck — APPROVED** [review]
Thanks for finding and fixing this!

**Pull request merged**